packages/coinstac-client-core/test/sub-api/project-service.js

```diff
@@ -59,14 +59,51 @@ test('ProjectService#setMetaContents errors', t => {
   const consortiumId = 'test-consortium';
   const projectId = 'test-project';
 
-  const dbGetStub = sinon.stub();
+  const dbGetSpy = sinon.spy();
+
+  // Sinon doesn't support dynamic stub responses based on args. Do it manually.
+  function dbGetStub(name) {
```

Comment:
You can’t do:

```js
const myStub = sinon.stub(function(name) {
  if (name === 'something-interesting') {
    return 'something-even-more-interesting';
  }
  return 'something-uninteresting';
});
```

…and then use `myStub` somewhere.

Also, this test is _super_ confusing to follow: it depends on carefully timed stub call counts. Any suggestions on improving?